war/src/main/resources/images/ionicons/ellipse-outline.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" class="ionicon" viewBox="0 0 512 512"><title>Ellipse</title><circle cx="256" cy="256" r="192" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="32"/></svg>
```

Comment:
How can we use the other icons? So we need to manually add them one by one? Wouldn't it make more sense to include all (like in the Font Awesome plugin)?

Reply:
It'd involve copying the icons from ionicons to this folder, wanted to avoid that in this PR as it'd balloon this PR up in size.

Reply:
The icons are currently part of resources folder (and not below the webapp folder). How can we replace icons like [in actions](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/jenkins/management/ConfigureLink.java) with these new ionicon icons?

Reply:
We can't yet, it wasn't part of this feature.

likely the l:icon tag needs to be able to detect its an ionicon and load it, possibly another flag or other way